core/templates/dev/head/dashboard/dashboard.html

```diff
@@ -0,0 +1,241 @@
+                        Private
+                      </span>
+                    </span>
+                  </div>
+                </div>
+                <div class="oppia-dashboard-tile-contents">
+                  <span class="oppia-dashboard-tile-image-container" style="position: absolute;">
+                    <img class="oppia-dashboard-tile-image" ng-src="<[exploration.thumbnail_icon_url]>" style="background-color: <[exploration.thumbnail_bg_color]>" ng-class="{'oppia-dashboard-tile-image-private': exploration.status === 'private'}">
+                  </span>
+
+                  <div class="oppia-dashboard-tile-details">
+                    <div class="oppia-dashboard-tile-first-row">
+                      <span class="oppia-dashboard-tile-title">
+                        <span ng-if="exploration.title"><[exploration.title]></span>
+                        <span ng-if="!exploration.title">Untitled</span>
+                      </span>
+                      <span style="font-weight: bold;">&middot;</span>
+                      <span style="color: #888; font-size: 0.9em;" class="protractor-test-exploration-objective">
+                        <span ng-if="exploration.objective">
+                          <[exploration.objective | truncateAndCapitalize: 105]>
+                        </span>
+                        <span ng-if="!exploration.objective">
+                          <em>No objective specified.</em>
+                        </span>
+                      </span>
+                    </div>
+
+                    <div class="oppia-dashboard-tile-second-row">
+                      <span style="color: #888">
+                        <strong><[exploration.category]></strong>
+                        <span style="font-weight: bold;" ng-if="exploration.category">&middot;</span>
+                        Last updated <[getLocaleAbbreviatedDatetimeString(exploration.last_updated)]>
+                      </span>
+
+                      <span ng-if="exploration.status === 'publicized'">
+                        <span style="font-weight: bold;">&middot;</span>
+                        <span style="color: #888">Featured</span>
+                      </span>
+
+                      <span ng-if="getAverageRating(exploration.ratings)">
+                        <span style="font-weight: bold;">&middot;</span>
+                        <span style="color: #888">
+                          Average rating <[getAverageRating(exploration.ratings) | number:1]>
+                        </span>
+                      </span>
+                    </div>
+
+                  </div>
+                </div>
+              </a>
+            </md-card>
+          </li>
+        </ul>
+      </div>
+
+      <div ng-if="activeTab === 'myCollections'">
+        <div ng-if="collectionsList.length === 0" class="oppia-dashboard-empty-text">
+          <em>There are no collections to display.</em>
```

Comment:
Is this even possible to show up? 'My Collections' only seems to appear if you have created a collection.